### DISCUSSION TITLE
Add ConnectCom Fiberpatchpanel 72 Port LC/APC

### DISCUSSION THREAD

**Discussion by optical-337** [netbox-community/devicetype-library#2375] [General]
Hello together,
can some1 add my device-template to the library? 
I want to share my work with fiber-components with the community but im not a real it guy - just a user that likes documentation...

Its a new manufacturer named ConnectCom (big one in germany for fiber material)

Please import it to the library (there will follow some more from this manufacturer)
Thanks a lot :)

the device:
---
manufacturer: ConnectCom
model: CCM Patchpanel 1HE SLITE HD Right
slug: ccm-patchpanel-1he-slite-hd-r
description: 36xLCAPC-D GN, 72x9-125 Set DIN
part_number: '#951797'
u_height: 1.0
is_full_depth: true
airflow: passive
comments: 'Connector type A: LC-APC connector, Standards: IEC 61754-20, TIA 604-10-A, Insertion loss: typ. ≤ 0.12 dB - 97% ≤ 0.25dB (Grade B), Return loss: typ. > 85 dB - min. 65 dB, Ferrule material: all-ceramic ferrule (1.25 mm)'
front_image: true
rear_image: true
is_powered: false
front-ports:
  - name: Front Port 1
    type: lc-apc
    rear_port: Rear Port 1
    rear_port_position: 1
  - name: Front Port 2
    type: lc-apc
    rear_port: Rear Port 2
    rear_port_position: 1
  - name: Front Port 3
    type: lc-apc
    rear_port: Rear Port 3
    rear_port_position: 1
  - name: Front Port 4
    type: lc-apc
    rear_port: Rear Port 4
    rear_port_position: 1
  - name: Front Port 5
    type: lc-apc
    rear_port: Rear Port 5
    rear_port_position: 1
  - name: Front Port 6
    type: lc-apc
    rear_port: Rear Port 6
    rear_port_position: 1
  - name: Front Port 7
    type: lc-apc
    rear_port: Rear Port 7
    rear_port_position: 1
  - name: Front Port 8
    type: lc-apc
    rear_port: Rear Port 8
    rear_port_position: 1
  - name: Front Port 9
    type: lc-apc
    rear_port: Rear Port 9
    rear_port_position: 1
  - name: Front Port 10
    type: lc-apc
    rear_port: Rear Port 10
    rear_port_position: 1
  - name: Front Port 11
    type: lc-apc
    rear_port: Rear Port 11
    rear_port_position: 1
  - name: Front Port 12
    type: lc-apc
    rear_port: Rear Port 12
    rear_port_position: 1
  - name: Front Port 13
    type: lc-apc
    rear_port: Rear Port 13
    rear_port_position: 1
  - name: Front Port 14
    type: lc-apc
    rear_port: Rear Port 14
    rear_port_position: 1
  - name: Front Port 15
    type: lc-apc
    rear_port: Rear Port 15
    rear_port_position: 1
  - name: Front Port 16
    type: lc-apc
    rear_port: Rear Port 16
    rear_port_position: 1
  - name: Front Port 17
    type: lc-apc
    rear_port: Rear Port 17
    rear_port_position: 1
  - name: Front Port 18
    type: lc-apc
    rear_port: Rear Port 18
    rear_port_position: 1
  - name: Front Port 19
    type: lc-apc
    rear_port: Rear Port 19
    rear_port_position: 1
  - name: Front Port 20
    type: lc-apc
    rear_port: Rear Port 20
    rear_port_position: 1
  - name: Front Port 21
    type: lc-apc
    rear_port: Rear Port 21
    rear_port_position: 1
  - name: Front Port 22
    type: lc-apc
    rear_port: Rear Port 22
    rear_port_position: 1
  - name: Front Port 23
    type: lc-apc
    rear_port: Rear Port 23
    rear_port_position: 1
  - name: Front Port 24
    type: lc-apc
    rear_port: Rear Port 24
    rear_port_position: 1
  - name: Front Port 25
    type: lc-apc
    rear_port: Rear Port 25
    rear_port_position: 1
  - name: Front Port 26
    type: lc-apc
    rear_port: Rear Port 26
    rear_port_position: 1
  - name: Front Port 27
    type: lc-apc
    rear_port: Rear Port 27
    rear_port_position: 1
  - name: Front Port 28
    type: lc-apc
    rear_port: Rear Port 28
    rear_port_position: 1
  - name: Front Port 29
    type: lc-apc
    rear_port: Rear Port 29
    rear_port_position: 1
  - name: Front Port 30
    type: lc-apc
    rear_port: Rear Port 30
    rear_port_position: 1
  - name: Front Port 31
    type: lc-apc
    rear_port: Rear Port 31
    rear_port_position: 1
  - name: Front Port 32
    type: lc-apc
    rear_port: Rear Port 32
    rear_port_position: 1
  - name: Front Port 33
    type: lc-apc
    rear_port: Rear Port 33
    rear_port_position: 1
  - name: Front Port 34
    type: lc-apc
    rear_port: Rear Port 34
    rear_port_position: 1
  - name: Front Port 35
    type: lc-apc
    rear_port: Rear Port 35
    rear_port_position: 1
  - name: Front Port 36
    type: lc-apc
    rear_port: Rear Port 36
    rear_port_position: 1
  - name: Front Port 37
    type: lc-apc
    rear_port: Rear Port 37
    rear_port_position: 1
  - name: Front Port 38
    type: lc-apc
    rear_port: Rear Port 38
    rear_port_position: 1
  - name: Front Port 39
    type: lc-apc
    rear_port: Rear Port 39
    rear_port_position: 1
  - name: Front Port 40
    type: lc-apc
    rear_port: Rear Port 40
    rear_port_position: 1
  - name: Front Port 41
    type: lc-apc
    rear_port: Rear Port 41
    rear_port_position: 1
  - name: Front Port 42
    type: lc-apc
    rear_port: Rear Port 42
    rear_port_position: 1
  - name: Front Port 43
    type: lc-apc
    rear_port: Rear Port 43
    rear_port_position: 1
  - name: Front Port 44
    type: lc-apc
    rear_port: Rear Port 44
    rear_port_position: 1
  - name: Front Port 45
    type: lc-apc
    rear_port: Rear Port 45
    rear_port_position: 1
  - name: Front Port 46
    type: lc-apc
    rear_port: Rear Port 46
    rear_port_position: 1
  - name: Front Port 47
    type: lc-apc
    rear_port: Rear Port 47
    rear_port_position: 1
  - name: Front Port 48
    type: lc-apc
    rear_port: Rear Port 48
    rear_port_position: 1
  - name: Front Port 49
    type: lc-apc
    rear_port: Rear Port 49
    rear_port_position: 1
  - name: Front Port 50
    type: lc-apc
    rear_port: Rear Port 50
    rear_port_position: 1
  - name: Front Port 51
    type: lc-apc
    rear_port: Rear Port 51
    rear_port_position: 1
  - name: Front Port 52
    type: lc-apc
    rear_port: Rear Port 52
    rear_port_position: 1
  - name: Front Port 53
    type: lc-apc
    rear_port: Rear Port 53
    rear_port_position: 1
  - name: Front Port 54
    type: lc-apc
    rear_port: Rear Port 54
    rear_port_position: 1
  - name: Front Port 55
    type: lc-apc
    rear_port: Rear Port 55
    rear_port_position: 1
  - name: Front Port 56
    type: lc-apc
    rear_port: Rear Port 56
    rear_port_position: 1
  - name: Front Port 57
    type: lc-apc
    rear_port: Rear Port 57
    rear_port_position: 1
  - name: Front Port 58
    type: lc-apc
    rear_port: Rear Port 58
    rear_port_position: 1
  - name: Front Port 59
    type: lc-apc
    rear_port: Rear Port 59
    rear_port_position: 1
  - name: Front Port 60
    type: lc-apc
    rear_port: Rear Port 60
    rear_port_position: 1
  - name: Front Port 61
    type: lc-apc
    rear_port: Rear Port 61
    rear_port_position: 1
  - name: Front Port 62
    type: lc-apc
    rear_port: Rear Port 62
    rear_port_position: 1
  - name: Front Port 63
    type: lc-apc
    rear_port: Rear Port 63
    rear_port_position: 1
  - name: Front Port 64
    type: lc-apc
    rear_port: Rear Port 64
    rear_port_position: 1
  - name: Front Port 65
    type: lc-apc
    rear_port: Rear Port 65
    rear_port_position: 1
  - name: Front Port 66
    type: lc-apc
    rear_port: Rear Port 66
    rear_port_position: 1
  - name: Front Port 67
    type: lc-apc
    rear_port: Rear Port 67
    rear_port_position: 1
  - name: Front Port 68
    type: lc-apc
    rear_port: Rear Port 68
    rear_port_position: 1
  - name: Front Port 69
    type: lc-apc
    rear_port: Rear Port 69
    rear_port_position: 1
  - name: Front Port 70
    type: lc-apc
    rear_port: Rear Port 70
    rear_port_position: 1
  - name: Front Port 71
    type: lc-apc
    rear_port: Rear Port 71
    rear_port_position: 1
  - name: Front Port 72
    type: lc-apc
    rear_port: Rear Port 72
    rear_port_position: 1
rear-ports:
  - name: Rear Port 1
    type: splice
    positions: 1
  - name: Rear Port 2
    type: splice
    positions: 1
  - name: Rear Port 3
    type: splice
    positions: 1
  - name: Rear Port 4
    type: splice
    positions: 1
  - name: Rear Port 5
    type: splice
    positions: 1
  - name: Rear Port 6
    type: splice
    positions: 1
  - name: Rear Port 7
    type: splice
    positions: 1
  - name: Rear Port 8
    type: splice
    positions: 1
  - name: Rear Port 9
    type: splice
    positions: 1
  - name: Rear Port 10
    type: splice
    positions: 1
  - name: Rear Port 11
    type: splice
    positions: 1
  - name: Rear Port 12
    type: splice
    positions: 1
  - name: Rear Port 13
    type: splice
    position: 1
  - name: Rear Port 14
    type: splice
    position: 1
  - name: Rear Port 15
    type: splice
    position: 1
  - name: Rear Port 16
    type: splice
    position: 1
  - name: Rear Port 17
    type: splice
    position: 1
  - name: Rear Port 18
    type: splice
    position: 1
  - name: Rear Port 19
    type: splice
    position: 1
  - name: Rear Port 20
    type: splice
    position: 1
  - name: Rear Port 21
    type: splice
    position: 1
  - name: Rear Port 22
    type: splice
    position: 1
  - name: Rear Port 23
    type: splice
    position: 1
  - name: Rear Port 24
    type: splice
    position: 1
  - name: Rear Port 25
    type: splice
    position: 1
  - name: Rear Port 26
    type: splice
    position: 1
  - name: Rear Port 27
    type: splice
    position: 1
  - name: Rear Port 28
    type: splice
    position: 1
  - name: Rear Port 29
    type: splice
    position: 1
  - name: Rear Port 30
    type: splice
    position: 1
  - name: Rear Port 31
    type: splice
    position: 1
  - name: Rear Port 32
    type: splice
    position: 1
  - name: Rear Port 33
    type: splice
    position: 1
  - name: Rear Port 34
    type: splice
    position: 1
  - name: Rear Port 35
    type: splice
    position: 1
  - name: Rear Port 36
    type: splice
    position: 1
  - name: Rear Port 37
    type: splice
    position: 1
  - name: Rear Port 38
    type: splice
    position: 1
  - name: Rear Port 39
    type: splice
    position: 1
  - name: Rear Port 40
    type: splice
    position: 1
  - name: Rear Port 41
    type: splice
    position: 1
  - name: Rear Port 42
    type: splice
    position: 1
  - name: Rear Port 43
    type: splice
    position: 1
  - name: Rear Port 44
    type: splice
    position: 1
  - name: Rear Port 45
    type: splice
    position: 1
  - name: Rear Port 46
    type: splice
    position: 1
  - name: Rear Port 47
    type: splice
    position: 1
  - name: Rear Port 48
    type: splice
    position: 1
  - name: Rear Port 49
    type: splice
    position: 1
  - name: Rear Port 50
    type: splice
    position: 1
  - name: Rear Port 51
    type: splice
    position: 1
  - name: Rear Port 52
    type: splice
    position: 1
  - name: Rear Port 53
    type: splice
    position: 1
  - name: Rear Port 54
    type: splice
    position: 1
  - name: Rear Port 55
    type: splice
    position: 1
  - name: Rear Port 56
    type: splice
    position: 1
  - name: Rear Port 57
    type: splice
    position: 1
  - name: Rear Port 58
    type: splice
    position: 1
  - name: Rear Port 59
    type: splice
    position: 1
  - name: Rear Port 60
    type: splice
    position: 1
  - name: Rear Port 61
    type: splice
    position: 1
  - name: Rear Port 62
    type: splice
    position: 1
  - name: Rear Port 63
    type: splice
    position: 1
  - name: Rear Port 64
    type: splice
    position: 1
  - name: Rear Port 65
    type: splice
    position: 1
  - name: Rear Port 66
    type: splice
    position: 1
  - name: Rear Port 67
    type: splice
    position: 1
  - name: Rear Port 68
    type: splice
    position: 1
  - name: Rear Port 69
    type: splice
    position: 1
  - name: Rear Port 70
    type: splice
    position: 1
  - name: Rear Port 71
    type: splice
    position: 1
  - name: Rear Port 72
    type: splice
    position: 1

**Comment by harryajc** [comment]
Hi @optical-337

There is a guide [here](https://github.com/netbox-community/devicetype-library/blob/master/CONTRIBUTING.md) on how to create a PR. Then it can be reviewed.

**Comment by phillf** [comment]
@optical-337,

- is ConnectCom an actual brand/manufacturer?
- Do you have a datasheet or specsheet?
- Do you have a cropped front image of this device?

Wondering if this should just be added as a generic.

**Reply by optical-337** [reply]
hey phillf,
- yes it is a german manufacturer here is the url: https://www.connectcom.de/de/
- datasheet: https://www.connectcom.de/includes/PIM/Documents/[951797_DE.pdf](https://www.connectcom.de/includes/PIM/Documents/951797_DE.pdf)
- Picture:
![ccm-1he-slite-panel-hd-72p_r](https://github.com/user-attachments/assets/3430af2f-d0e0-4eb2-8a9f-f12598379700)

thanks for your interest in this :)
